MV.Wildebeest.Core/source/core/java/co/mv/wb/Wildebeest.java

```diff
@@ -178,21 +172,37 @@ public static WildebeestApiBuilder wildebeestApi(
 	// Global Functions
 	//
 
+	// implement from label to id here nad remove this comment when you are done
 	public static State stateForId(
 		Resource resource,
-		UUID stateId)
+		String stateId)
 	{
 		if (resource == null) { throw new IllegalArgumentException("resource cannot be null"); }
 		if (stateId == null) { throw new IllegalArgumentException("stateId cannot be null"); }
 
+		final String isUUIDmatcher = "[a-zA-Z0-9][a-zA-Z0-9\\-\\_ ]+[a-zA-Z0-9]";
```

Comment:
It would be nice to separate the UUID check out into a static helper function - could go in Utils